plugin.go

```diff
@@ -36,6 +39,17 @@ type LintPlugin struct {
 	aspectplugin.Base
 	// This plugin will store some state from the Build Events for use at the end of the build.
 	command_line.CommandLine
+	// Helper to parse our config section
+	yamlUnmarshalStrict    func(in []byte, out interface{}) (err error)
```

Comment:
Why inject it like this instead of just using the library?

Reply:
I dunno, I copied from https://github.com/aspect-build/plugin-augment-error/blob/main/plugin.go#L30
Maybe it's meant to be for testability?

Reply:
I'd drop it if you had no specific reason. Can always add it back if actually needed 🤷

Reply:
Okay I simplified it to empty "constructor" call